fem/linearform.hpp

```diff
@@ -185,10 +199,22 @@ class LinearForm : public Vector
        corresponding pointer (to Array<int>) will be NULL. */
    Array<Array<int>*> *GetFLFI_Marker() { return &boundary_face_integs_marker; }
 
+   /// Set the desired assembly level.
+   /** Valid choices are:
+
+       - AssemblyLevel::LEGACY (default)
+       - AssemblyLevel::FULL
+
+       This method must be called before assembly. */
```

Comment:
> This method must be called before assembly.

Not strictly true — if it's not called, it will just default to `LEGACY`

Reply:
Addressed in https://github.com/mfem/mfem/pull/3235/commits/c60263c1fd9ecaa9a437346bbb69e172939d7fa1